bin/tlint

```diff
@@ -16,6 +16,10 @@ foreach (
     }
 }
 
+if (file_exists($appAutoloader = getcwd() . '/vendor/autoload.php')) {
```

Comment:
Can we add this to the above array of other autoloader paths (line 10)?

e.g. `[ __DIR__ . '/vendor/autoload.php', getcwd() . '/vendor/autoload.php']`

Reply:
I originally was going to do that, but my concern is if there’s a composer dependency that `tlint` requires, but the app does not, it will fail. Either we would need to commit to exclusively depending on libraries that Laravel core depends on, or else we need to import both autoloaders.

Reply:
I think I see, this is used to load the custom Linters from the Laravel app (and in turn loads all the apps dependencies)?

Reply:
Yep. The autoloader on line 10 loads the `tlint` dependencies. The new autoloader loads any app dependencies. You kind-of want both, unless you decide that `tlint` can never depend on something that Laravel core doesn’t, in which case we’d have to update the composer dependencies and then adjust this code.

Reply:
I believe the only time this comes into play is when there are custom linters right? So this seems fine 👍

Reply:
Yeah. It only applies in the case where `tlint` is installed globally and you’re using custom linters.